internal/mock/mock_fs/fs.go

```diff
@@ -0,0 +1,25 @@
+package mock_fs
```

Comment:
Since it is under `mock` folder, `package mockfs` or `package fs` is more appropriate.

Reply:
Renamed to mockfs.